custom/sensors/SparkEncoder.java

```diff
@@ -0,0 +1,134 @@
+package org.usfirst.frc4904.standard.custom.sensors;
+
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import org.usfirst.frc4904.standard.Util;
+import edu.wpi.first.wpilibj.PIDSourceType;
+
+public class SparkEncoder implements CustomEncoder {
+    protected final CANEncoder encoder;
+    protected PIDSourceType pidSource;
+    protected double distancePerPulse;
```

Comment:
SparkEncoders return data in rotations (or rotations per minute) [page 13](http://www.revrobotics.com/content/sw/max/sw-docs/SPARK-MAX-Java-API-Offline.pdf). You should change from using pulses to rotations